buildAndReleaseTask/scascan.ts

```diff
@@ -82,7 +82,7 @@ async function run(): Promise<void> {
                 python3.arg('install');
                 python3.arg('--upgrade');
                 python3.arg('pip');
-                python3.arg('junitparser');
+                python3.arg('junitparser==1.6.3');
```

Comment:
anyway to not hard code this?

Reply:
@gattjoe What is the process space when Azure runs the plugin? Can we put a requirements.txt in buildAndReleaseTask and call pip install -r requirements.txt instead?

It will still be packaged and static to a commit, but at least it is externalized to one requirements.txt file and follows python best practice.

Reply:
I can refactor the code to use requirements.txt

Reply:
> anyway to not hard code this?

It's probably best practice to provide the version, but I don't know the impact when it isn't specified.